website/views.py

```diff
@@ -98,11 +97,6 @@ def _render_nodes(nodes, auth=None, show_path=False):
 
 @collect_auth
 def index(auth):
-    """Redirect to dashboard if user is logged in, else show homepage.
-
-    """
-    if auth.user:
-        return redirect(web_url_for('home'))
     return {}
```

Comment:
Yes! Here's where the `OsfWebRenderer__index` string is [computed](https://github.com/CenterForOpenScience/osf.io/blob/cef7a1f89871bf624473b556ff70157b6701a1bf/framework/routing/__init__.py#L159-L162)—we do it ourselves in `framework` (it's not a Flask or Werkzeug thing).